face_detection_masking/test.py

```diff
@@ -33,3 +33,53 @@ def test_upload_processed_images():
 
     # If all assertions passed, then return test pass
     return "Test passed"
```

Comment:
QQ: do we have to `return "Test passed"`?

Reply:
No, you don't have to. I just want to make it responsive.